### DISCUSSION TITLE
Where's node_modules?

### DISCUSSION THREAD

**Discussion by francis-jjk** [kriasoft/react-starter-kit#2017] [Q&A]
IDE: VS Code

**Comment by koistya** [comment]
@francis-jjk template is using Yarn with PnP — an alternative to they Node.js modules are installed locally and resolved, you can learn more about this approach here:

https://yarnpkg.com/features/pnp

It is possible to disable it, by setting `nodeLinker: node-modules` in `.yarnrc.yml` file and reinstalling all the dependencies (`yarn install`), as a legacy approach with `node_modules` folder.

https://github.com/kriasoft/react-starter-kit/blob/f589b535ddc29193d8be4aeed076f30c0895d691/.yarnrc.yml#L3

Also, it is possible to hide some files or folders in VS Code settings here:

https://github.com/kriasoft/react-starter-kit/blob/f589b535ddc29193d8be4aeed076f30c0895d691/.vscode/settings.json#L33
